.github/workflows/check_for_new_releases.yaml

```diff
@@ -0,0 +1,71 @@
+name: Check for new releases
+
+on:
+  schedule:
+    - cron: '0 0 */14 * *'  # Runs every two weeks on Sunday at midnight
+  workflow_dispatch:
+
+jobs:
+  create_issue:
+    name: Create new release issue
+    runs-on: ubuntu-latest
+    env:
+      SYNAPSE_CURRENT_VERSION: v1.101.1
+    permissions:
+      issues: write
+    steps:
+      - id: synapse
+        uses: pozetroninc/github-action-get-latest-release@master
+        with:
+          owner: element-hq
+          repo: synapse
+          excludes: prerelease, draft
+
+      - uses: actions/checkout@v4.1.1
```

Comment:
Maybe getting the current Synapse version from rockcraft.yaml instead of a variable in this yaml file?

Reply:
Excellent suggestion, done!